docs/content/InputField.mdx

```diff
@@ -17,7 +17,7 @@ Use [FormControl](/FormControl) instead.
 
 ### Basic
 
-```jsx live
+```jsx live deprecated
```

Comment:
`deprecated` won't work until https://github.com/primer/react/pull/1887 is merged

Reply:
it's merged ✅